programmers/20210503/짝수와-홀수.js

```diff
@@ -0,0 +1,9 @@
+function solution(num) {
+    if((num % 2) == 0){
+        var answer = "Even";
+    }
+    else{
+        var answer = "Odd";
+    }
+    return answer;
+}
```

Comment:
삼항연산자를 이용해서 한 줄로도 풀 수 있습니다.
자바스크립트에서 0은 true 1은 false기 때문에 이 특성을 이용해서
num % 2가 0 즉, true라면 "Odd"를 반환하고 1이라면 false "Even"을 반환해라
```suggestion
function solution(num) {
  return num % 2 ? "Odd" : "Even";
}
```